unity-client/Assets/UnityGLTF/Scripts/Loader/FileLoader.cs

```diff
@@ -38,6 +38,10 @@ private IEnumerator LoadFileStream(string rootPath, string fileToLoad)
             if (!File.Exists(pathToLoad))
             {
                 Debug.LogError($"Buffer file not found ({pathToLoad}) -- {fileToLoad}");
+
+                if(DCL.DataStore.i.ABConversorRunning && !DCL.DataStore.i.ABConversorGLTFMissingDependencies.Contains(fileToLoad))
```

Comment:
GLTF importer shouldn't know about `ABConverter`. We should expose this behaviour through an additional setting on `GLTFImporter` or something like that.

Reply:
To add a more solid proposal:

* We can add a new parameter to the Editor GLTF Importer (just add another public field to `GLTFImporter` class)
* On `ABConverter.Core`, after copying the files, we can retrieve the importer distance using `AssetImporter.GetAtPath` ([docs](https://docs.unity3d.com/ScriptReference/AssetImporter.GetAtPath.html)). We can then set the parameter of the importer. On some cases, `AssetImporter.GetAtPath` wouldn't do the trick because we have to wait a frame, if this is the case, we can just try to make the setting static. This second approach is a bit messy though.
* On `GLTFImporter` we can set an additional parameter to the created `GLTFSceneImporter` using our new parameter ([this code](https://github.com/decentraland/explorer/blob/f05cb55cc33c717923f3545c3ee3bff84288a1e6/unity-client/Assets/UnityGLTF/Scripts/Editor/GLTFImporter.cs#L499-L504))

Reply:
The refactor to detect dependencies first and add them to the rawContents to be downloaded made this change unneeded so I reverted it.